src/Compilers/Core/Portable/Operations/ControlFlowRegionKind.cs

```diff
@@ -18,8 +15,9 @@ public enum ControlFlowRegionKind
         Root,
 
         /// <summary>
-        /// Region with the only purpose to represent the life-time of locals and methods (local functions, lambdas).
-        /// PROTOTYPE(dataflow): We should clearly explain what "life-time" refers to here, or use a different term.
+        /// Region with the only purpose to represent the life-time of locals and nested methods (local functions, lambdas).
+        /// Lifetime for a local symbol represents the region within which the local allocation is valid and can be referenced.
```

Comment:
>and can be referenced. [](start = 105, length = 22)

I don't believe is is precisely true, correct? For example, this is one region:
```
int a = 1;
int b = 2;
```
But I couldn't have declared a as `int a = b;` because b isn't assigned yet.

Reply:
Yes, but that is true due to a restriction imposed by the language rather than the runtime. Let me know if you have a better and simpler way to convey the semantics.

---
In reply to: [193208400](https://github.com/dotnet/roslyn/pull/27477#discussion_r193208400) [](ancestors = 193208400)

Reply:
I think we should probably fall back to the spec on this. The spec says:
> The lifetime of a local variable is the portion of program execution during which storage is guaranteed to be reserved for it. 
https://github.com/dotnet/csharplang/blob/master/spec/variables.md#local-variables